tests/console/snapper_cleanup.pm

```diff
@@ -24,6 +24,11 @@ sub snapper_cleanup {
     my $free_space      = script_output("$btrfs_fs_usage | sed -n '7p' | awk -F ' ' '{print\$3}'");
     my $excl_free_space = ($fs_size / 2);
 
+    diag 'Check if we have enough free space to create snapshots';
+    if ($free_space < $excl_free_space) {
+        die "The test most probably will fail, FS has less free space then expected";
```

Comment:
* single ticks

and better write 'The filesystem has leess free space than this tests expects, the test will most likely fail'